README.md

```diff
@@ -61,6 +61,10 @@ To customise the platform app:
 - Create a directory somewhere outside this repository where you will put your customisation files.
 - Add an environment variable called `$CUSTOMISATIONS_DIR` that points to the directory created in the step above.
 - Inside your customisation directory you can overwrite or add new files following the same directory structure as `src/public`.
+- Make sure you have [yarn](https://classic.yarnpkg.com/en/docs/install) installed.
+  Any version below 2.0, but no older than 1.0,
+  will defer to the appropriate version while working in this repository.
+  Run `yarn install` in the repository to download dependencies.
```

Comment:
Maybe we should include the `--frozen-lockfile` option here, to ensure that anyone who isn't actively updating the dependencies gets the versions that the developers tested.

Reply:
Never mind, after taking another look at [the docs](https://classic.yarnpkg.com/en/docs/cli/install#toc-yarn-install-frozen-lockfile) I think that only adds another way for the build process to fail, which is good for CI but wouldn't give the best impression to people who've just checked out the codebase for the first time.